src/test/java/com/artipie/npm/proxy/NpmProxyITCase.java

```diff
@@ -78,19 +69,18 @@
     private static int listenPort;
 
     /**
-     * Mock server test container.
-     * @todo #1:120m Use real NPM repo instead of mocks.
-     *  Take a look at https://github.com/verdaccio/verdaccio for example
+     * Node test container.
      */
     @org.testcontainers.junit.jupiter.Container
-    private final MockServerContainer mockctner = new MockServerContainer();
+    private final NodeContainer npmcnter = new NodeContainer()
+        .withCommand("tail", "-f", "/dev/null");
 
     /**
-     * Node test container.
+     * Verdaccio test container.
      */
     @org.testcontainers.junit.jupiter.Container
-    private final NodeContainer npmcnter = new NodeContainer()
-        .withCommand("tail", "-f", "/dev/null");
+    private final VerdaccioContainer verdaccio = new VerdaccioContainer()
```

Comment:
@paulodamaso I don't see the value of the `VerdaccioContainer` class. Can't we just use `GenericContainer`?

Reply:
@carlosmiranda Actually we have a bug (?) in `testcontainer` (see https://github.com/testcontainers/testcontainers-java/issues/238). So that's the solution I found to it (the diamond operator didn't worked for me)

Reply:
@paulodamaso right. Can we add a comment describing that in the javadoc for this class?